sentry_sdk/integrations/aws_lambda.py

```diff
@@ -277,10 +281,8 @@ def event_processor(event, hint, start_time=start_time):
         if "headers" in aws_event:
             request["headers"] = _filter_headers(aws_event["headers"])
 
-        if aws_event.get("body", None):
-            # Unfortunately couldn't find a way to get structured body from AWS
-            # event. Meaning every body is unstructured to us.
-            request["data"] = AnnotatedValue("", {"rem": [["!raw", "x", 0, 0]]})
+        if "body" in aws_event:
+            request["data"] = aws_event.get("body", "")
```

Comment:
@untitaker I made this change to display the request body appear on Sentry Dashboard as shown below. This change was discussed with AJ.

![req_body](https://user-images.githubusercontent.com/7957539/94348771-a8910680-005c-11eb-8567-661a7e4ef256.png)

Reply:
Please restore the old behavior when the client option `send_default_pii` is in its default `False`. If the user set `send_default_pii=True` we can do what you propose.

Also please restore the code comment.

The reason for this is that we want to be safe by default: the request body can contain a lot of sensitive data and setting sendDefaultPii is our way of having the user say "I accept the risks".

Reply:
@untitaker you mean that essentially, I move this section of code I've written to the if section for send_default_pii part and revert the changes here.

Reply:
sorry I mean

```
if send_default_pii:
    new_code
else:
    old_code
```

Reply:
Okay, I'll make these changes asap.